src/EditorFeatures/Core/Implementation/NavigationBar/NavigationBarController_ModelComputation.cs

```diff
@@ -25,13 +26,15 @@ internal partial class NavigationBarController
         /// The computation of the last model.
         /// </summary>
         private Task<NavigationBarModel> _modelTask;
-        private NavigationBarModel _lastCompletedModel;
```

Comment:
remove this entirely.  very dangerous/confusign to have the task and the state.  now, the state is only *in* the task, and gets passed along through that to anyone that needs it.